lib/versions.js

```diff
@@ -1,20 +1,25 @@
-const semver = require('semver')
+const inc = require('semver/functions/inc')
+const major = require('semver/functions/major')
+const minor = require('semver/functions/minor')
+const patch = require('semver/functions/patch')
+const parse = require('semver/functions/parse')
+const coerce = require('semver/functions/coerce')
```

Comment:
Same question here, is this required to make it work? If so, do you know why that is?
I'd prefer to keep just one import because `semver.coerce` is a lot clearer in its intent than just `coerce` and if it just takes a `require('semver')` to break the build then I fear that will happen sooner rather than later. Please correct me if I'm wrong though :)

Reply:
`require('semver')` does not load functions at the same time it adds a lot of code to ncc output.
you can use `require('semver/preload')` but again it adds even more to the ncc output.

Reply:
Feel free to test it out locally :)